code/game/machinery/doors/door.dm

```diff
@@ -262,11 +262,6 @@
 	operating = 0
 	air_update_turf(1)
 	update_freelok_sight()
-
-	//I shall not add a check every x ticks if a door has closed over some fire.
-	var/obj/effect/hotspot/fire = locate() in loc

```

Comment:
It really is; it has no reason to exist under LINDA; it was specifically introduced because of a ZAS fire refactor and is meant for that system; there's no reason for it to continue existing now.